components/dashboard/provider/ProviderReviewsTab.tsx

```
@@ -0,0 +1,131 @@
+'use client';
+
+import { Card, CardHeader, CardTitle, CardDescription, CardContent, CardFooter } from "@/components/ui/card";
+import { Button } from "@/components/ui/button";
+import { ArrowRight } from "lucide-react";
+import Link from "next/link";
+import { useProviderReviews } from "@/hooks/providers/use-provider-reviews";
+import { useProviderMetrics } from "@/hooks/providers/use-provider-metrics";
+
+export function ProviderReviewsTab() {
+  const { data: metrics } = useProviderMetrics();
+  const { data: reviews, isLoading } = useProviderReviews();
```

ESLint

[error] 'reviews' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)